apidoc-plugin/build.gradle

```diff
@@ -6,25 +6,37 @@ import org.mozilla.apilint.Config
 
 buildDir "${topobjdir}/apidoc-plugin"
 
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.4.1'
```

Comment:
Explain why you need this?  I care because I'm intending to use https://github.com/wupdigital/android-maven-publish for publishing https://github.com/mozilla/application-services.  I came to that because the dcendents one is geared to the old style `maven` plugin and not the new-style `maven-publish` plugin.  I see you are using `maven-publish`, so I'm confused.  In fact, you don't appear to be publishing an Android publication at all (i.e., an AAR), so do you really need this?

Reply:
As discussed in chat, this is not needed! Thanks for catching it.